gene_expression_scatter_plot/js/main.js

```diff
@@ -14,6 +14,7 @@ var groups = new Array();
 var attrMatchs = new Array();
 var tissueMatchs = new Array();
 var detailMatchs = new Array();
+var dic = new Array();  //定义一个字典
```

Comment:
What is the variable `dic ` for?

It seems that `dic `has never been used.

Hui

Reply:
dic是一个字典变量，用来存储字典的。condition_specific_correlation函数是用来得出字典的，所以我把这个函数返回值赋值给dic，但是原先的代码里，groups储存的内容与dic相似，所以并没有使用dic，还是继续使用groups。

Reply:
> dic是一个字典变量，用来存储字典的。condition_specific_correlation函数是用来得出字典的，所以我把这个函数返回值赋值给dic，但是原先的代码里，groups储存的内容与dic相似，所以并没有使用dic，还是继续使用groups。

OK.  Delete `dic ` if it is not used.  Otherwise people like me would wonder what the use of it is.

**Hui**

Reply:
已改